configs/components/oifs/oifs.yaml

```diff
@@ -101,10 +107,23 @@ compile_infos:
             git-repository: https://gitlab.dkrz.de/ec-earth/oifs-43r3.git
             install_bins: make/esm/oifs/bin/oifs
           43r3-awi:
-            branch: master
+            requires:
+            - oasis3mct-4.0-oifsdeck
+            branch: master # or oifs-deck???
```

Comment:
We should use https://gitlab.dkrz.de/ec-earth/oifs-43r3/-/tags/awicm-3-deck
Master will keep moving on.